pallets/msa/src/lib.rs

```diff
@@ -112,13 +130,48 @@ pub mod pallet {
 		pub fn create(origin: OriginFor<T>) -> DispatchResult {
 			let who = ensure_signed(origin)?;
 
-			let (identifier, _) = Self::create_account(who.clone())?;
+			let (identifier, _) = Self::create_account(who.clone(), EMPTY_FUNCTION)?;
 
 			Self::deposit_event(Event::MsaCreated { msa_id: identifier, key: who });
 
 			Ok(())
 		}
 
+		#[pallet::weight(T::WeightInfo::create(10_000))]
```

Comment:
this is going to be replaced with fresh calculated weights in the next PR which will be dedicated to benchmarks!